### ISSUE TITLE
2.0.7 - Key not found in dictionary, RD error

### ISSUE THREAD

**Issue by peterennis** [rubberduck-vba/Rubberduck#2130]
This occurred when doing compact and repair.
Also saw it when closing Access.
![capture51](https://cloud.githubusercontent.com/assets/140737/17112841/a088f9a0-525c-11e6-848a-6b0831228bf1.PNG)
![capture50](https://cloud.githubusercontent.com/assets/140737/17112840/a0854f08-525c-11e6-95ad-7408eebca6d8.PNG)
![capture49](https://cloud.githubusercontent.com/assets/140737/17112839/a0853734-525c-11e6-80f0-8f47e296d736.PNG)
[rubberduck.zip](https://github.com/rubberduck-vba/Rubberduck/files/382231/rubberduck.zip)

**Comment by daFreeMan** [comment]
At least a partial duplicate of #2081. The 2nd RTE (80131577) is the exact same number I reported there.

**Comment by Hosch250** [comment]
Are you on 2.0.7?

**Comment by peterennis** [comment]
Yes on 2.0.7
Restart access shows this:
![capture52](https://cloud.githubusercontent.com/assets/140737/17113147/2e8e451a-525e-11e6-9fe4-4df9069680d6.PNG)

**Comment by daFreeMan** [comment]
I've been getting a lot of that today. I just marked it down to "Monday", but it may be deeper than that.

**Comment by peterennis** [comment]
Open Access. Go to VBA. It starts resolving dependencies.
Back to Access and Compact and Repair.
Error.
New log here.
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/382269/RubberduckLog.txt)

**Comment by daFreeMan** [comment]
Not sure if it helps, but I just got a "Stopped Working" from Excel while code was executing.

Not sure if you'll see anything useful from the log, but here it is 
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/382303/RubberduckLog.txt)

**Comment by peterennis** [comment]
I restarted the PC and repeated the steps. It seem repeatable to me.
I recall reading about a plan to make parsing on demand vs. automatic and that would
seem to apply here. If I start it and do something else before it finished then I broke it.
If I do something without being aware this process is running then the blame goes elsewhere.
![capture53](https://cloud.githubusercontent.com/assets/140737/17113822/2e2697fa-5261-11e6-83ad-973158065a01.PNG)
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/382289/RubberduckLog.txt)
